packages/core/src/libraries/mocks.ts

```diff
@@ -1,39 +0,0 @@
-import path from "path";
-import { Files } from "../node-utils";
-import { SharedSliceContent } from "@prismicio/types-internal/lib/documents/widgets/slices";
-
-export function createPathToMock({
-  path: filePath,
-  from,
-  sliceName,
-}: {
-  path: string;
-  from: string;
-  sliceName: string;
-}): string {
-  return path.join(filePath, from, sliceName, "mocks.json");
-}
-
-export function resolvePathsToMock({
```

Comment:
no-longer needed?

Reply:
no, I simplified the code, you'll see when reviewing, it's no longer necessary

Reply:
good